config/plugins/visualizations/hyphyvision/src/.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I guess this can be part of the main gitignore?

Reply:
Interesting, looks like it already is.

```
{base}dave@gaius:~/github-repos/hyphy-vision-rebase $ grep node_modules .gitignore 
# JS, Local node_modules, and bower_components directories
node_modules
```